exercises/acronym/example.rs

```diff
@@ -0,0 +1,20 @@
+pub fn abbreviate(phrase: &str) -> String {
+    phrase.split(|c: char| c.is_whitespace() || !c.is_alphanumeric())
+          .flat_map(|word| split_camel(word))
+          .filter(|word| !word.is_empty())
```

Comment:
Do you think this line is unclear as an alternative?
```rust
.filter_map(|word| word.chars().next().map(|c| c.to_uppercase().collect::<String>()))
```
It deals with the possibility of an empty word, but it ends up being kind of verbose. I didn't realize that `char.to_uppercase()` returns an iterator rather than just a `char`.